src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -123,14 +123,14 @@ def _get_docker_status_and_version(ignore_errors, yes):
 
 # Get current CLI version
 def _get_cli_version():
-    acr_component_name = "azure-cli-acr"
-    acr_cli_version = "not found"
     from pkg_resources import working_set
-    for component in list(working_set):
-        if component.key == acr_component_name:
-            acr_cli_version = component.version
 
-    print('ACR CLI version: {}'.format(acr_cli_version))
+    # working_set.by_key is a dictionary with component names as key
+    cli_component_name = "azure-cli"
+    cli_version = working_set.by_key[cli_component_name].version \
+        if cli_component_name in working_set.by_key else "not found"
+
+    print('ACR CLI version: {}'.format(cli_version))
```

Comment:
This needs to print to stderr. 
https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#general-patterns

Reply:
@tjprescott Thanks for pointing that out. New changes have been pushed.

Reply:
Hi @tjprescott , this command is used by customers to get general information on the environment being used and connectivity to a registry. The command output consists of general info of the environment being used, including this, that's why we are using print (here and in other pieces of the command as well)

Reply:
@tjprescott @ni-reis 

CI build scripts seem to be failing while directly using `print('.....', file=sys.stderr)` due to incompatibility with Py 2.7. 

Apparently, an import statement `from __future__ import print_function` needs to be added to make it compatible with both Py2.7 and Py3+.  For instance, this file [uses the same fix here](https://github.com/Azure/azure-cli/blob/dev/scripts/dev_setup.py#L8).

As Nicholas mentioned before, maybe we should reconsider printing to `stderr` in the first place since this is just a general information being shown to the users. Any suggestions?

Reply:
The fact that it is printed for informational purposes only is exactly why it should be printed to stderr. Anything printed to stdout is expected to be something one might pipe to another command. Informational messages don't fall into that category.

Reply:
I see, thanks for clarifying! 
From our side, we want to make sure that the default is that this info will show in the command line (similar to a command output), i.e. we don't want this to be printed to another file that the user would have to check to see the output. Will we get this behavior printing to stderr?

Reply:
The info will display in the command prompt in white font. But if the user redirects stdout to file or pipes stdout to another command, it will not pass those info message along to that.

Reply:
I see. Thanks!
@hkuadithya , can you please make those changes and test then?